src/style.css

```diff
@@ -0,0 +1,108 @@
+@font-face {
+  font-family: "Avenir";
+  font-style: normal;
+  font-weight: 300;
+  src: url(./fonts/Avenir.ttc);
+}
+
+@font-face {
+  font-family: "Avenir-Heavy";
+  font-weight: bold;
+  src: url(fonts/Avenir-Heavy.ttf);
+
+}
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  font-family: "Avenir", sans-serif;
+  font-size: 16px;
+  font-weight: 300;
+  line-height: normal;
+  color: #3d4e61;
+}
+
+.wrap-big {
+  height: 70px;
+}
+
+.wrap-small {
+  height: 42px;
+}
+
+.search {
+  height: 100%;
+  position: relative;
+  box-sizing: border-box;
+  display: flex;
+  align-items: center;
+  font-family: "Avenir", sans-serif;
+  margin-top: 20px;
+  font-weight: 600;
+}
+
+.search__icon {
+  position: absolute;
+}
+
+.search__icon--big {
+  top: 25px;
+  bottom: 26px;
+  left: 26px;
+  width: 19px;
+  height: 19px;
+}
+
+.search__icon--small {
+  top: 15px;
+  left: 13px;
+  width: 11px;
+  height: 11px;
+}
+
+.search__input {
+  color: #3d4e61;
+  font-family: "Avenir", sans-serif;
+  box-sizing: border-box;
+  width: 100%;
+  height: 70px;
+  border-radius: 4px;
+  border: 1px solid #e1e7ed;
```

Comment:
Pay attention to CSS properties order
https://9elements.com/css-rule-order/
Also, would be great to create variables for colors and other main values